src/software/sensor_fusion/sensor_fusion.cpp

```diff
@@ -156,15 +157,15 @@ void SensorFusion::updateWorld(const SSL_DetectionFrame &ssl_detection_frame)
     {
         for (auto &detection : ball_detections)
         {
-            invert(detection);
+            detection = invert(detection);
```

Comment:
assuming this fixes something, can we add a test for this?

Reply:
added test